homeassistant/components/rainforest_raven/manifest.json

```
@@ -0,0 +1,26 @@
+{
+  "domain": "rainforest_raven",
+  "name": "Rainforest RAVEn",
+  "codeowners": ["@cottsay"],
+  "config_flow": true,
+  "dependencies": ["usb"],
+  "documentation": "https://www.home-assistant.io/integrations/rainforest_raven",
+  "iot_class": "local_polling",
+  "requirements": ["aioraven==0.4.0", "iso4217==1.11.20220401"],
```

```suggestion
  "requirements": ["aioraven==0.4.0"],
```

Your integration code is not using `iso4217`